test/cli/environment/log-file/env

```diff
@@ -0,0 +1 @@
+KBC_VERSION_CHECK=false
```

Comment:
Sice je tato ENV nastavena v CI, ale ako sme to upravili, aby sa do testov nedostavali externe ENVs, tak nastal tento problem.